engine/core/IgeEntity.js

```diff
@@ -2085,6 +2045,8 @@ var IgeEntity = IgeObject.extend({
 				}
 
 				this.tween.start(effect.tween, angle);
+			// this needs its own PR for fix in master
+			// adding too many variables if we add to PR 334
```

Comment:
not sure what this refers to, why it should stay in code and end up on master, and why it is not written in trello?